src/mutmap/test/data/dng.vcf

```diff
@@ -0,0 +1,125 @@
+##fileformat=VCFv4.2
```

Comment:
Remove test data.  Testdata is in the testdata repository and is downloaded when `make test` runs.